pkg/util/grpcclient/register.go

```diff
@@ -0,0 +1,66 @@
+package grpcclient
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/kubeedge/mappers-go/config"
+	dmiapi "github.com/kubeedge/mappers-go/pkg/apis/dmi/v1"
+	"github.com/kubeedge/mappers-go/pkg/common"
+	"google.golang.org/grpc"
+)
+
+// RegisterMapper if withData is true, edgecore will send device and model list.
+func RegisterMapper(cfg *config.Config, withData bool) ([]*dmiapi.Device, []*dmiapi.DeviceModel, error) {
+	// 连接grpc服务器
```

Comment:
please comment with English

Reply:
or the Chinese comments in this file can be all removed.

Reply:
done